src/modules/julia.rs

```diff
@@ -0,0 +1,558 @@
+//! Pretend to update julia packages
+use async_trait::async_trait;
+use instant::Instant;
+use rand::prelude::*;
+use std::fmt::Display;
+use yansi::Paint;
+
+use crate::args::AppConfig;
+use crate::data::JULIA_PACKAGES_LIST;
+use crate::generators::gen_hex_string;
+use crate::io::{csleep, cursor_up, dprint, erase_line, newline};
+use crate::modules::Module;
+
+// Redefine a more efficient `crate::io::print` function that doesn't type one character at a time.
+async fn print<S: Into<String>>(s: S) {
+    #[cfg(target_arch = "wasm32")]
+    {
+        write_to_xterm(s.into().as_str());
+    }
+
+    #[cfg(not(target_arch = "wasm32"))]
+    {
+        use std::io::stdout;
+        use std::io::Write;
+        print!("{}", s.into());
+        stdout().flush().unwrap();
+    }
+}
```

Comment:
Yeah, this was a long time coming anyway. I added this to master directly. Stuff like this should not really be a per-module thing. Thanks for the hint. You can now remove this. :)

Reply:
I agree that it should be made globally available, but I didn't want to touch other files in order not to give the impression that I was revolutionizing the whole repository. I'll rebase on this change then. :)